bedrock/firefox/templates/firefox/privacy/book.html

```diff
@@ -42,7 +42,7 @@ <h2 class="c-sub-navigation-title">{{ ftl('sub-navigation-privacy') }}</h2>
 <section class="mzp-c-hero">
   <div class="mzp-l-content">
     <div class="mzp-c-hero-body">
-      <img src="{{ static('img/firefox/privacy/book/privacy-eyes.png') }}" width="200" height="200" alt="">
+      <img src="{{ static('img/firefox/privacy/book/privacy-eyes.png') }}" width="200" height="200" alt="" loading="lazy">
```

Comment:
No need to add `loading="lazy"` to images that appear above the fold.

Reply:
Indeed, I will remove it.